cookbook/import_export/product-export.rst

```diff
@@ -0,0 +1,200 @@
+Understand the CSV Product Export
+=================================
+
+We'll discuss here how the csv product export works.
+
+It's a good start to understand the overall architecture and how to re-use or replace some parts.
+
+.. note::
+
+  The import part has been widely re-worked in 1.4, please note that the export part remains the same than in 1.3 (all classes and services are in the BaseConnectorBundle).
+
+Definition of the Job
+---------------------
+
+The product export is defined in ``src\Pim\Bundle\BaseConnectorBundle\Resources\config\batch_jobs.yml``.
+
+.. code-block:: yaml
+
+    connector:
+        name: Akeneo CSV Connector
+        jobs:
+            csv_product_export:
+                title: pim_base_connector.jobs.csv_product_export.title
+                type:  export
+                steps:
+                    export:
+                        title:     pim_base_connector.jobs.csv_product_export.export.title
+                        services:
+                            reader:    pim_base_connector.reader.doctrine.product
+                            processor: pim_base_connector.processor.product_to_flat_array
+                            writer:    pim_base_connector.writer.file.csv_product
+                        parameters:
+                            batch_size: 10
+
+With the ``type`` parameter, we can see that this job is an export.
+
+We can also count a single step named ``export``.
```

Comment:
Maybe we should have a different name for the step as it may be thought to be related to the "type" although it could be named "foo_export" for example.